packages/components/src/TimelineFlightDetail/TimelineFlightDetail.js

```diff
@@ -13,9 +13,9 @@ import CardHeader from './components/CardHeader';
 type Props = {|
   +additionalInfo: $PropertyType<AdditionalInfoProps, 'additionalInfo'>,
   +carrier: {|
-    +code: string,
-    +name: string,
-    +type?: 'airline' | 'bus' | 'train',
+    code: string,
```

Comment:
Why without `+`?

Reply:
Because the current types are as follows in universal-components:
https://github.com/kiwicom/universal-components/blob/a252171631194d56ae42d1785f620cd6116a4c61/src/CarrierLogo/CarrierLogoTypes.js#L3